src/rollup/types.d.ts

```diff
@@ -73,15 +75,15 @@ export interface PluginContext {
 	emitAsset: (name: string, source?: string | Buffer) => string;
 	setAssetSource: (assetId: string, source: string | Buffer) => void;
 	getAssetFileName: (assetId: string) => string;
-	warn(warning: RollupWarning, pos?: { line: number; column: number }): void;
-	error(err: RollupError, pos?: { line: number; column: number }): void;
+	warn(warning: RollupWarning | string, pos?: { line: number; column: number }): void;
+	error(err: RollupError | string, pos?: { line: number; column: number }): void;
 }
 
 export type ResolveIdHook = (
 	this: PluginContext,
 	id: string,
 	parent: string
-) => Promise<string | boolean | void> | string | boolean | void;
+) => Promise<string | boolean | void | null> | string | boolean | void | null;
```

Comment:
I honestly thought `null` was included in `void`?

Reply:
Unfortunately not.
Open the following URL and check the `strictNullChecks` option.

https://www.typescriptlang.org/play/#src=function%20foo()%3A%20void%20%7B%0A%20%20%20%20return%20null%3B%0A%7D